bok_choy/browser.py

```diff
@@ -206,8 +206,7 @@ def _local_browser_class(browser_name):
     """
 
     # Log name of local browser
-    LOGGER.info(
-        "Using local browser: {0} [Default is firefox]".format(browser_name))
+    LOGGER.info("Using local browser: %s [Default is firefox]", browser_name)
```

Comment:
In the platform, I thought we favored using `.format()`.   Will bok-choy not follow that?